repair_type/models/stock_move.py

```diff
@@ -35,3 +35,16 @@ def _get_repair_locations(self, repair_line_type, repair_id=False):
                     res[1],
                 )
         return res
+
+    def _compute_location_id(self):
+        res = True
+        for move in self:
+            if move.repair_line_type and move.repair_id:
+                location_src, location_dest = move._get_repair_locations(
+                    move.repair_line_type
+                )
+                move.location_id = location_src
+                move.location_dest_id = location_dest
```

Comment:
shouldn't the location dest id be changed in the compute method compute_location_dest_id from standard?

Reply:
:thinking:  yes, let me check

Reply:
You are right! I did the change :)